components/automate-cli/pkg/adminmgmt/adminmgmt_test.go

```diff
@@ -335,113 +309,6 @@ func TestAddAdminUserToTeam(t *testing.T) {
-				Policies: []*authz_resp.Policy{
-					{
-						Id:       "wrong-id",
-						Subjects: []string{authz_constants.LocalAdminsTeamSubject, "some:other:subject"},
-					},
-				},
-			}, nil
-		}
-
-		serverMocks.AuthzMock.CreatePolicyFunc = createDefaultAdminPolicy
-
-		foundPolicy, createdNewPolicy, err := adminmgmt.UpdateV1AdminsPolicyIfNeeded(ctx, apiClient, false)
-		require.NoError(t, err)
-		assert.False(t, foundPolicy)
-		assert.True(t, createdNewPolicy)
-	})
-
-	t.Run("when the original admins policy exists and contains the admins team it returns true, false, nil", func(t *testing.T) {
-		serverMocks.AuthzMock.ListPoliciesFunc = func(
-			context.Context, *authz_req.ListPoliciesReq) (*authz_resp.ListPoliciesResp, error) {
-
-			return &authz_resp.ListPoliciesResp{
-				Policies: []*authz_resp.Policy{
-					{
-						Id:       authz_constants_v1.AdminPolicyID,
-						Subjects: []string{authz_constants.LocalAdminsTeamSubject, "some:other:subject"},
-					},
-				},
-			}, nil
-		}
-
-		serverMocks.AuthzMock.CreatePolicyFunc = func(
-			context.Context, *authz_req.CreatePolicyReq) (*authz_resp.CreatePolicyResp, error) {
-			return nil, errors.New("unexpected call")
-		}
-
-		foundPolicy, createdNewPolicy, err := adminmgmt.UpdateV1AdminsPolicyIfNeeded(ctx, apiClient, false)
-		require.NoError(t, err)
-		assert.True(t, foundPolicy)
-		assert.False(t, createdNewPolicy)
-	})
-
-	t.Run("when the original admins policy exists and does not contain the admins team it returns true, true, nil", func(t *testing.T) {
-		serverMocks.AuthzMock.ListPoliciesFunc = func(
-			context.Context, *authz_req.ListPoliciesReq) (*authz_resp.ListPoliciesResp, error) {
-
-			return &authz_resp.ListPoliciesResp{
-				Policies: []*authz_resp.Policy{
-					{
-						Id:       authz_constants_v1.AdminPolicyID,
-						Subjects: []string{"some:other:subject"},
-					},
-				},
-			}, nil
-		}
-		serverMocks.AuthzMock.CreatePolicyFunc = createDefaultAdminPolicy
-
-		foundPolicy, createdNewPolicy, err := adminmgmt.UpdateV1AdminsPolicyIfNeeded(ctx, apiClient, false)
-		require.NoError(t, err)
-		assert.True(t, foundPolicy)
-		assert.True(t, createdNewPolicy)
-	})
-
-	t.Run("when the original admins policy exists and does not contain the admins team it returns true, true, nil", func(t *testing.T) {
-		serverMocks.AuthzMock.ListPoliciesFunc = func(
-			context.Context, *authz_req.ListPoliciesReq) (*authz_resp.ListPoliciesResp, error) {
-
-			return &authz_resp.ListPoliciesResp{
-				Policies: []*authz_resp.Policy{
-					{
-						Id:       authz_constants_v1.AdminPolicyID,
-						Subjects: []string{"some:other:subject"},
-					},
-				},
-			}, nil
-		}
-		serverMocks.AuthzMock.CreatePolicyFunc = createDefaultAdminPolicy
-
-		foundPolicy, createdNewPolicy, err := adminmgmt.UpdateV1AdminsPolicyIfNeeded(ctx, apiClient, false)
-		require.NoError(t, err)
-		assert.True(t, foundPolicy)
-		assert.True(t, createdNewPolicy)
-	})
-}
-
 func TestUpdateV2AdminsPolicyIfNeeded(t *testing.T) {
```

Comment:
Match suggested function name change mentioned earlier:
```suggestion
func TestUpdateAdminPolicyIfNeeded(t *testing.T) {
```